main/BUILD

```diff
@@ -149,6 +149,7 @@ r_binary(
         ":train_classifier_g",
         ":train_polite_score",
         "//src:convo_word_freq_diff",
+        "//src:find_SE_words",
```

Comment:
You've added a dependency here, and so I expect that feed_data.R would call this.

Reply:
That's correct. Lemme write an issue for this.